test/test_mypy.py

```diff
@@ -87,6 +104,66 @@ def test_bulk_write(self) -> None:
         result = self.coll.bulk_write(requests)
         self.assertTrue(result.acknowledged)
 
+    def test_command(self) -> None:
+        result = self.client.admin.command("ping")
+        items = result.items()
+
+    def test_list_collections(self) -> None:
+        cursor = self.client.test.list_collections()
+        value = cursor.next()
+        items = value.items()
+
+    def test_list_databases(self) -> None:
+        cursor = self.client.list_databases()
+        value = cursor.next()
+        value.items()
+
+    def test_default_document_type(self) -> None:
+        client = rs_or_single_client()
+        coll = client.test.test
+        doc = {"my": "doc"}
+        coll.insert_one(doc)
+        retreived = coll.find_one({"_id": doc["_id"]})
+        assert retreived is not None
+        retreived["a"] = 1
+
+    def test_explicit_document_type(self) -> None:
```

Comment:
This test looks the same as test_default_document_type

Reply:
Updated